app/services/by_team_project_fetcher_service.rb

```diff
@@ -46,7 +46,7 @@ def users
         name: user.full_name,
         email: user.email,
         id: user.id,
-        conversion_count: Conversion::Project.assigned_to(user).count
+        conversion_count: Conversion::Project.in_progress.assigned_to(user).count
```

Comment:
`not_completed` might be more appropriate here as `in_progess` assumes assigned? Not a deal breaker though.